src/components/SelectionControls/CheckboxLabel.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import Checkbox from "@material-ui/core/Checkbox";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+
+class CheckboxLabel extends React.Component {
+  public state = {
+    checked: true
+  };
+
+  public handleChange = () => (
```

Comment:
It seems to me `handleChange` should just be `(event) => void`, not `() => (event) => void`. Then line 23 has to be `onChange={this.handleChange}`.